relay-server/src/actors/processor.rs

```diff
@@ -2351,9 +2351,19 @@ impl EnvelopeProcessorService {
             SamplingResult::Drop(rule_ids)
                 if state.event_type() == Some(EventType::Transaction) =>
             {
-                state
-                    .managed_envelope
-                    .reject(Outcome::FilteredSampling(rule_ids.clone()));
+                let feature_enabled = false;
```

Comment:
TODO: Add a new feature to the enum below, and check it here.

https://github.com/getsentry/relay/blob/1adb61adc8c19322cacef75203195690f06575ae/relay-dynamic-config/src/feature.rs#L7